src/invert.js

```diff
@@ -0,0 +1,12 @@
+'use strict'
+
+/**
+ * Inverts the key-value pairs of an object.
+ *
+ * @param {object} object - The object to invert.
+ * @returns {object} - The new object with inverted key-value pairs.
+ */
+const invert = (object) =>
+  Object.fromEntries(Object.entries(object).map(([key, value]) => [value, key]))
```

Comment:
This approach has more lines of code but is more efficient as it has just a single loop
```suggestion
const invert = (object) => {
  const newObject = {}
  for (const key in object) {
    newObject[object[key]] = key
  }
  return newObject
}
```